scripts/factory-hooks/post-settings-php/memcache.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Example implementation of ACSF post-settings-php hook.
+ *
+ * @see https://docs.acquia.com/site-factory/tiers/paas/workflow/hooks
+ */
+
+// Use ACSF feature flag to indicate if a site should use memcache backend
+if ($is_ah_env &&
```

Comment:
We already have logic to include `memcache.settings.php` here: https://github.com/acquia/blt/blob/9.x/settings/cache.settings.php#L38

Can we move this logic over to cache.settings.php so it's all in one place instead of split between ACE / ACSF?

Reply:
blt settings are included in the pre-settings-php hook and ACSF has config in settings.php which needs to be overridden by the blt memcache.settings.php settings, hence the post-settings.php hook